src/deadline/houdini_submitter/python/deadline_cloud_for_houdini/submitter.py

```diff
@@ -20,87 +20,16 @@
 from deadline.job_attachments.models import JobAttachmentS3Settings
 
 from .queue_parameters import update_queue_parameters, get_queue_parameter_values_as_openjd
+from .assets import _get_hip_file, _get_asset_references, _parse_files
 from ._version import version
 
 import hou
 
-IGNORE_REF_VALUES = ("opdef:", "oplib:", "temp:", "op:")
-IGNORE_REF_PARMS = (
```

Comment:
Gah, unfortunately we previously marked these as public in a public module. To prevent a needless breaking change let's just import these from the assets module and ignore them for now.

The longer term fix to prevent this type of stuff from coming up is clarifying our public interfaces on the integrations since they're applications. ie. Is it just how we launch the application?